core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1092,7 +1092,7 @@ private void processWorkerAddedBlocks(MasterWorkerInfo workerInfo,
   private void processWorkerOrphanedBlocks(MasterWorkerInfo workerInfo) {
     for (long block : workerInfo.getBlocks()) {
       if (!mBlockStore.getBlock(block).isPresent()) {
-        LOG.info("Requesting delete for orphaned block: {} from worker {}.", block,
+        LOG.debug("Requesting delete for orphaned block: {} from worker {}.", block,
```

Comment:
have one info log indicating the total number of orphaned block for this worker?